testament/testament.nim

```diff
@@ -515,7 +714,8 @@ proc addResult(r: var TResults, param: ReportParams, givenSpec: ptr TSpec) =
 
   logToBackend(param)
 
-  # TODO DOC what is this
+  # REFACTOR what is this, no string interpolation until the data is
```

Comment:
test results aren't the only thing to note/log, so it's not that straightfoward.

Reply:
![image](https://user-images.githubusercontent.com/20562256/205459014-da638e06-cb58-4929-8714-4b51380adf1f.png)

I want to do batches/logging part like this

Reply:
and to be honest I couldn't really understand what `.data` does, I will look into it more, maybe this will throw a wrench in this data flow model